dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/event/TaskResultEventHandler.java

```
@@ -115,11 +120,16 @@ public void handleTaskEvent(TaskEvent taskEvent) throws TaskEventHandleError, Ta
     }
 
     public void sendAckToWorker(TaskEvent taskEvent) {
-        ITaskInstanceExecutionEventAckListener instanceExecutionEventAckListener =
-                SingletonJdkDynamicRpcClientProxyFactory
-                        .getProxyClient(taskEvent.getWorkerAddress(), ITaskInstanceExecutionEventAckListener.class);
-        instanceExecutionEventAckListener.handleTaskInstanceExecutionFinishEventAck(
-                TaskInstanceExecutionFinishEventAck.success(taskEvent.getTaskInstanceId()));
+        try {
+            ITaskInstanceExecutionEventAckListener instanceExecutionEventAckListener =
+                    SingletonJdkDynamicRpcClientProxyFactory
+                            .getProxyClient(taskEvent.getWorkerAddress(), ITaskInstanceExecutionEventAckListener.class);
+            instanceExecutionEventAckListener.handleTaskInstanceExecutionFinishEventAck(
+                    TaskInstanceExecutionFinishEventAck.success(taskEvent.getTaskInstanceId()));
+        } catch (Exception e) {
+            // master ignore the exception, worker will retry to send this TaskEventType.RESULT event again.
+            log.error("send ack to worker error, taskInstanceId: {}", taskEvent.getTaskInstanceId(), e);
```

```suggestion
            log.warn("send ack to worker error, taskInstanceId: {}", taskEvent.getTaskInstanceId(), e);
```

---

If we ignore this exception, it's better to change to warn.